websites/D/Doodstreams/metadata.json

```diff
@@ -11,7 +11,11 @@
 	},
 	"url": "doods.pro",
 	"regExp": "((ww(w)?([0-9])?[.])?dood(s)?[.][a-z]*)|((ww(w)?([0-9])?[.])?doodstream[.][a-z]*([.][a-z]*)?)",
-	"version": "1.0.2",
+	"matches": [
+		"*://*.doodstreams.*/*",
+		"*://*.doods.*/*"
```

Comment:
This glob is not a valid [match pattern](https://developer.chrome.com/docs/extensions/develop/concepts/match-patterns).
> If you use a wildcard in the host pattern, it must be the first or only character

Reply:
wdym

Reply:
So what would be a solution?

Reply:
If you know all the tlds that the site uses, you can add them all in a list

Reply:
> If you know all the tlds that the site uses, you can add multiple of them in a list

It seems almost random.